lib/camp3/request.rb

```diff
@@ -8,7 +8,7 @@ class Request
     include HTTParty
     include Logging
     format :json
-    headers 'Accept' => 'application/json'
+    headers 'Accept' => 'application/json', 'Content-Type' => 'application/json'
```

Comment:
Assumed that `format :json` was already doing this, but for some reason, a `400` response is returned if the explicit **Content-Type** header is missing

Reply:
According to [making a request](https://github.com/basecamp/bc3-api#making-a-request), **Content-Type** is required for requests that create new resources, like `add_comments`. That's a great finding.